src/transformers/integrations/peft.py

```diff
@@ -77,7 +83,7 @@ def load_adapter(
         Requires peft as a backend to load the adapter weights.
 
         Args:
-            peft_model_id (`str`):
+            peft_model_id (`str` or dictionary of `torch.Tensor`):
                 The identifier of the model to look for on the Hub, or a local path to the saved adapter config file
                 and adapter weights.
```

Comment:
Doc needs to be updated if this can be a ` state_dict`. The name of the argument is not intuitive but I guess wee need to keep it for Backward compatibility? Otherwise would rather have a new arg

Reply:
I think for BC it's fine as the feature is quite new

Reply:
Not really if it was part of the release 😅